### PR TITLE
added small deatils

### PR DESCRIPTION
**Pull request by MadPemaDog**
there is no achor in index to link to this

### CODE REVIEW

File: src/login.html [hunk 1]
```diff
@@ -2,8 +2,40 @@
...
+                max-width: 700px;
+                max-height: 700px;
+                margin: auto;
+                display: flex;
+                align-items: center;
+            }
+            .form {
+                height: 600px;
+                width: 75%;
+                margin: auto auto;
```
Comment (death-heaven) [inline src/login.html:19]: center the div

File: src/login.html [hunk 2]
```diff
@@ -2,8 +2,40 @@
...
+        </style>
     </head>
     <body>
-        hello world
+        <form action="">
+            <div class="form">
+                <label for="username">Username</label>
+                <input type="email" id="username" placeholder="johndoe@example.com" required>
+
+                <label for="password">Password</label>
```
Comment (death-heaven) [inline src/login.html:34]: this can be improve

**Review by death-heaven — CHANGES_REQUESTED** [review]

File: src/login.html (hunk 1 above)
Comment (death-heaven) [reply src/login.html:19]: centereds a div

File: src/login.html (hunk 2 above)
Comment (death-heaven) [reply src/login.html:34]: added anchir

File: src/login.html (hunk 1 above)
Comment (MadPemaDog) [reply src/login.html:19]: added anchor

**Review by death-heaven — APPROVED** [review]

**Pull request merged**